bin/pg_repack.c

```diff
@@ -84,6 +85,8 @@ const char *PROGRAM_VERSION = "unknown";
 	"  AND l.pid NOT IN (pg_backend_pid(), $1) " \
 	"  AND (l.virtualxid, l.virtualtransaction) <> ('1/1', '-1/0') " \
 	"  AND (a.application_name IS NULL OR a.application_name <> $2)" \
+	"  AND a.query !~* E'^\\\\s*vacuum\\\\s+' " \
```

Comment:
Note it's possible there may be no trailing space after 'vacuum' in the query text, e.g. `VACUUM(freeze)mytable;` or just `VACUUM;`.